MainApp/public/app/views/pages/admin/manageAdmin.html

```diff
@@ -0,0 +1,62 @@
+<div class="page-header">
+    <h1>Manage Admin</h1>
+</div>
+<div class="row" ng-controller="manageAdminCtrl">
+    <div class="col-lg-6">
+        <div class="panel panel-default">
+            <div class="panel-heading">
+                <h3 class="panel-title">Add New Admin</h3>
+            </div>
+            <div class="panel-body">
+                <form ng-submit="createNewAdmin();">
+                    <label>Email Id:</label>
+                    <input class="form-control" type="text" name="emailId" placeholder="Enter email"
```

Comment:
I'm not sure about this.
Did we have a requirement that the email id should always be an asu.edu email id?
If yes, validation would be needed for this input